fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -815,13 +815,9 @@ private void initAggregateBuiltins() {
                 Type.VARCHAR, Type.VARCHAR, true, false, false));
 
         for (Type t : Type.getSupportedTypes()) {
-            if (t.isNull()) {
-                continue; // NULL is handled through type promotion.
-            }
-            if (t.isChar()) {
-                continue; // promoted to STRING
-            }
-            if (t.isTime()) {
+            // null/char/time is handled through type promotion
+            // array/json/pseudo is not supported yet
```

Comment:
please add TODO, we could support array and json type later.

Reply:
ok